core/agents/agent_adapters.py

```
@@ -0,0 +1,312 @@
+"""
+Agent adapters for integrating specialized agents with the Wrenchai framework.
+
+These adapters provide compatibility layers to ensure specialized agents work correctly
+within the framework, especially for tool usage and state management.
+"""
+
+import logging
+from typing import Dict, Any, List, Callable, Optional, Union
+
+from .agent_state import agent_state_manager
```

Ruff (0.8.2)

`.agent_state.agent_state_manager` imported but unused

Remove unused import: `.agent_state.agent_state_manager`

(F401)